docs/app/Examples/collections/Grid/Types/GridDividedExample.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Divider, Image, Grid } from 'stardust'
+
+const image = <Image src='http://semantic-ui.com/images/wireframe/media-paragraph.png' />
+
+const GridDividedExample = () => (
+  <div>
+    <Grid column={3} divided>
+      <Grid.Row>
+        <Grid.Column>{image}</Grid.Column>
+        <Grid.Column>{image}</Grid.Column>
+        <Grid.Column>{image}</Grid.Column>
+      </Grid.Row>
+
+      <Grid.Row>
+        <Grid.Column>{image}</Grid.Column>
+        <Grid.Column>{image}</Grid.Column>
+        <Grid.Column>{image}</Grid.Column>
+      </Grid.Row>
+    </Grid>
+
+    <Divider />
```

Comment:
To keep examples clean and concise, let's use separate example files instead of a `<Divider />` between examples.  Then when adding the `<ComponentExample />` to the index.js for the related examples, just omit the `title` and `description`.  This way they'll appear all under one heading but have some whitespace between them and separate code toggle buttons.